scripts/build_widget.py

```diff
@@ -52,18 +64,42 @@ def main(argv):
         subprocess.check_call([args.yarn_path, "build:all"],
```

Comment:
I think you want to do the install at line 47, then if the use_local_changes is true, do those steps (all look correct), then resume the normal flow of build (line 63) and clean up (line 91, which is currently in an else block). The new automation should be very helpful, thank you for adding this script!

Reply:
The reason I didn't do that is because build:all didn't work. Only build did. For reasons I don't understand...